src/views/resources/Resources.tsx

```diff
@@ -0,0 +1,69 @@
+import Head from "next/head";
+import resources from "assets/data/resources.json";
+import aiML from "assets/icons/ai-ml.svg";
+import backend from "assets/icons/backend.svg";
+import database from "assets/icons/database-fill.svg";
+import deploy from "assets/icons/deploy.svg";
+import userInterface from "assets/icons/display.svg";
+import gaming from "assets/icons/gaming.svg";
+import tools from "assets/icons/tools.svg";
```

Comment:
Note: for future reference, might be nice to include the source of these icon assets as part of the commit message body.